_posts/2019-03-22-java-darmowa-czy-nie.md

```diff
@@ -0,0 +1,58 @@
+---
+layout:    post
+title:     "Java darmowa, czy nie?"
+date:      2019-03-28 08:00:00 +0100
+published: true
+author:    jwilczewski
+tags:
+    - java
+---
+> End of Public Updates for Oracle JDK 8
+> 
+> Oracle will not post further updates of Java SE 8 to its public download sites for commercial use after January 2019. Customers who need continued access to critical bug fixes and security fixes as well as general maintenance for Java SE 8 or previous versions can get long term support through Oracle Java SE Subscription or Oracle Java SE Desktop Subscription. For more information, and details on how to receive longer term support for Oracle JDK 8, please see the Oracle Java SE Support Roadmap.
+>
+> *[https://www.oracle.com/technetwork/java/javase/overview/index.html](https://www.oracle.com/technetwork/java/javase/overview/index.html)*
+
+15 stycznia tego roku światło dzienne ujrzał JDK 8u202 - ostatni darmowy update JDK 8. Darmowy do zastosowań komercyjnych. Wersja JDK 8 była pierwszą wersją LTS (Long-Term-Support) i była z nami od marca 2014 roku. Co dalej? Czy można zostać na JDK 8? Czy przesiadać się na kolejną wersję?
+
+## Oracle JDK vs OpenJDK
+
+W obecnej sytuacji szczególnego znaczenia nabierają różnice pomiędzy JDK releasowanym przez Oracle a OpenJDK.
+* [OpenJDK](http://openjdk.java.net/) jest projektem open source dostarczającym implementację  Java Platform, Standard Edition. Projekt działa od 2007 roku, a jednym z głównych kontrybutorów jest Oracle.
+* [Oracle JDK](https://www.oracle.com/technetwork/java/javase/overview/index.html) natomiast to implementacja Java Platform dostarczana i supportowana przez Oracle w ramach OTN (Oracle Technology Network).
```

Comment:
czy aby? implementacją jest OpenJDK a Oracle JDK to build/dystrybucja zawierająca implementację i narzędzia producenta?